pkgs/leak_tracker/lib/src/leak_tracking/leak_tracker_model.dart

```diff
@@ -86,6 +95,7 @@ class LeakTrackingConfiguration {
     this.checkPeriod = const Duration(seconds: 1),
     this.disposalTimeBuffer = const Duration(milliseconds: 100),
     this.leakDiagnosticConfig = const LeakDiagnosticConfig(),
+    this.gcCountBuffer = defaultGcCountBuffer,
```

Comment:
Same point here (if `LeakTrackingConfiguration` is public we should mention the addition of a new ctor param).